django/urls/converters.py

```diff
@@ -53,6 +56,18 @@ class PathConverter(StringConverter):
 
 
 def register_converter(converter, type_name):
+    if (
+        REGISTERED_CONVERTERS.get(type_name) is not None
+        or DEFAULT_CONVERTERS.get(type_name) is not None
```

Comment:
Use `in` for a faster and clearer check:

```suggestion
        type_name in REGISTERED_CONVERTERS
        or type_name in DEFAULT_CONVERTERS
```

[docs](https://docs.python.org/3/tutorial/datastructures.html#dictionaries):

> To check whether a single key is in the dictionary, use the in keyword.